src/components/pages/devices/flyouts/deviceNew/deviceNew.js

```diff
@@ -0,0 +1,411 @@
+                  </Radio>
+                  <Radio link={this.isGenerateIdLink} key={DeviceIdTypeOptions.generate.labelName} value={DeviceIdTypeOptions.generate.value}>
+                    {t(DeviceIdTypeOptions.generate.labelName)}
+                  </Radio>
+                </FormGroup>,
+                <FormGroup key="authType">
+                  <FormLabel key={AuthTypeOptions.labelName}>{t(AuthTypeOptions.labelName)}</FormLabel>
+                  <Radio link={this.authenticationTypeLink} key={AuthTypeOptions.symmetric.labelName} value={AuthTypeOptions.symmetric.value}>
+                    {t(AuthTypeOptions.symmetric.labelName)}
+                  </Radio>
+                  <Radio link={this.authenticationTypeLink} key={AuthTypeOptions.x509.labelName} value={AuthTypeOptions.x509.value}>
+                    {t(AuthTypeOptions.x509.labelName)}
+                  </Radio>
+                </FormGroup>,
+                <FormGroup key="authKeyType">
+                  <FormLabel key={AuthKeyTypeOptions.labelName}>{t(AuthKeyTypeOptions.labelName)}</FormLabel>
+                  <Radio link={this.isGenerateKeysLink} key={AuthKeyTypeOptions.generate.labelName} value={AuthKeyTypeOptions.generate.value} disabled={isX509}>
+                    {t(AuthKeyTypeOptions.generate.labelName)}
+                  </Radio>
+                  <Radio link={this.isGenerateKeysLink} key={AuthKeyTypeOptions.manual.labelName} value={AuthKeyTypeOptions.manual.value}>
+                    {t(AuthKeyTypeOptions.manual.labelName)}
+                  </Radio>
+                  <div className="sub-settings">
+                    <FormLabel>{isX509 ? t('devices.flyouts.new.authenticationKey.primaryThumbprint') : t('devices.flyouts.new.authenticationKey.primaryKey')}</FormLabel>
+                    <FormControl link={this.primaryKeyLink} disabled={isGenerateKeys} type="text" placeholder={t('devices.flyouts.new.authenticationKey.hint')} />
+                    <FormLabel>{t(isX509 ? t('devices.flyouts.new.authenticationKey.secondaryThumbprint') : 'devices.flyouts.new.authenticationKey.secondaryKey')}</FormLabel>
+                    <FormControl link={this.secondaryKeyLink} disabled={isGenerateKeys} type="text" placeholder={t('devices.flyouts.new.authenticationKey.hint')} />
+                  </div>
+                </FormGroup>
+              ]
+            }
+            <SummarySection title={t('devices.flyouts.new.summaryHeader')}>
+              <SummaryCount>{summaryCount}</SummaryCount>
+              <SectionDesc>{summaryMessage}</SectionDesc>
+              {this.state.isPending && <Indicator />}
+              {completedSuccessfully && <Svg className="summary-icon" path={svgs.apply} />}
+            </SummarySection>
+
+            {
+              error &&
+              <div className="devices-new-error">
```

Comment:
I don't think you need this div. You should be able to pass a unique class to the `ErrorMsg` itself.